java/rocksjni/portal.h

```diff
@@ -607,7 +607,7 @@ class WriteTypeJni {
     }
 
     // Get the DELETE enum field of org.rocksdb.WBWIRocksIterator.WriteType
-    static jobject DELETE(JNIEnv* env) {
+    static jobject DELETE_(JNIEnv* env) {
```

Comment:
There is some macro DELETE on Windows platform (then we have compile error). I can also use undef if you see it better.